ci/test_wheel_raft_dask.sh

```diff
@@ -11,7 +11,13 @@ RAPIDS_PY_WHEEL_NAME="raft_dask_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels
 RAPIDS_PY_WHEEL_NAME="pylibraft_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./local-pylibraft-dep
 python -m pip install --no-deps ./local-pylibraft-dep/pylibraft*.whl
 
-# echo to expand wildcard before adding `[extra]` requires for pip
-python -m pip install $(echo ./dist/raft_dask*.whl)[test]
+python -m pip install "raft_dask-${RAPIDS_PY_CUDA_SUFFIX}[test]>=0.0.0a0" --find-links dist/
 
-python -m pytest ./python/raft-dask/raft_dask/test
+# rapids-logger "pytest raft-dask"
+# python -m pytest ./python/raft-dask/raft_dask/test
+
+# rapids-logger "pytest raft-dask (ucx-py only)"
+# python -m pytest ./python/raft-dask/raft_dask/test --run_ucx
+
+rapids-logger "pytest raft-dask (ucxx only)"
+python -m pytest ./python/raft-dask/raft_dask/test --run_ucxx
```

Comment:
These flag names are a bit confusing. Do they really mean only run the ucx-py/ucxx tests? What happens if you don't pass either flag? Do _all_ tests run?

Reply:
Passing no flags run all Dask tests with `protocol="tcp"`, including those that are not specifically meant to test comms. With `--run_ucx`/`run_ucxx` _only_ tests specifically mean to test comms run with `protocol="ucx"`(UCX-Py)/`protocol="ucxx"`(UCXX)`. The reason for running them separately is that we need to create full Dask clusters with the respective comms configuration, setting clusters up take both time and, more importantly, memory, thus we cannot run all in one pass.